src/containers/search_result.jsx

```diff
@@ -89,8 +89,19 @@ class SearchResult extends Component {
     this.props.actions.changeColumnVisibility(column, checked);
   }
 
+  onSortMethodChange(value) {
+    this.props.actions.changeSortMethod(value);
+    const searchInputValue = this.props.searchInputValue;
```

Comment:
The value of search input can be gained from `this.props.query.term`. There is no need to pass this value through props.

Reply:
Finished